src/styles/style.css

```diff
@@ -290,4 +290,77 @@ body > div.sidebar-layout,
 .code-showcase code.copy-code {
   display: none;
   visibility: hidden;
+}
+
+/*
+ * Documentation links
+ */
+
+ .doc-links .stage-chip,
+ .doc-links ul,
+ .doc-links ul li {
+   display: inline-block;
+   margin: 0 1rem 0 0;
+ }
```

Comment:
This code can be slightly removed if we are moving the stage chip into the list

Reply:
The indentation here also looks off a little in comparison to the rest of the styles.